detox/e2e/support/server_api/client.js

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import axios from 'axios';
```

Comment:
curious why axios? We do use a polyfill  of `fetch` that runs RNFetchBlob under the hood, in any case using `fetch` alone should be good.

Reply:
The only reason is it's easier for axios to set global defaults, like when setting [MM token](https://github.com/mattermost/mattermost-mobile/pull/4535/files#diff-26bd05ece016185166520fa7aff89481R39), so that there's no need to handle it somewhere for succeeding API request after the user logged in. But very much open for change or suggestion on how to better handle it with `fetch` or `node-fetch`.

Reply:
we can keep it as is for now, if people start to confused the use of axios with fetch on the actual app, then we can look into it.